src/tides.f90

```diff
@@ -50,6 +50,11 @@ module tides
         procedure :: initTides
         procedure :: getTidesAcceleration
         procedure :: setTidesInitFlag
+        procedure :: getDelaunay_arg
+        procedure :: getDoodson_arg
+        procedure :: initFES2004
+        procedure :: getFES2004_corrections
```

Comment:
You can start to use snake_case for all functions and variables. NEPTUNE has been written using camelCase, but in FORTRAN we move to the snake_case convention. This means that you can rename your functions like:

- get_gelaunay_arg
- get_doodson_arg
etc.